package.json

```diff
@@ -2,10 +2,15 @@
   "name": "react-flexview",
   "version": "1.0.1",
   "description": "A react component to abstract over flexbox",
-  "main": "index.js",
+  "files": [
+    "lib",
```

Comment:
If we decide to replace `.npmignore` with `files` we should export also `src` as it's used in many of our projects.

I would do this in another issue anyway